package.json

```diff
@@ -33,8 +33,9 @@
   "homepage": "https://github.com/revdotcom/revai-node-sdk#readme",
   "dependencies": {
     "axios": "^0.18.0",
-    "express": "^4.16.4",
-    "form-data": "^2.3.3"
+    "form-data": "^2.3.3",
+    "stream": "0.0.2",
```

Comment:
if we are using nodejs streams then we shouldn't include this dependency to a third party stream library.